test/controllers/auctions_controller_test.rb

```diff
@@ -0,0 +1,76 @@
+require 'test_helper'
+require 'date'
+
+class AuctionsControllerTest < ActionDispatch::IntegrationTest
+  test 'should get /auctions endpoint' do
+    get '/auctions'
+    assert_response :success
+  end
+
+  test '/auctions should retrieve all auctions as json' do
+    auctions_to_create = [
+      {
+        title: 'Mustang',
+        description: '1968 Ford Mustang',
+        start_bid: 12000.00,
+        current_bid: 12000.00,
+        end_time: 10.days.from_now,
+      },
+      {
+        title: 'Fallen Empires Booster',
+        description: '3x MTG Fallen Empires booster packs',
+        start_bid: 10.00,
+        current_bid: 14.50,
+        end_time: 10.days.from_now,
+      }
+    ]
+
+    auctions_to_create.each do |auction| Auction.create(
+        title: auction[:title],
+        description: auction[:description],
+        start_bid: auction[:start_bid],
+        current_bid: auction[:current_bid],
+        end_time: auction[:end_time]
+      ) end
+
+    get '/auctions'
+    auction_response = JSON.parse(@response.body, symbolize_names: true)
+    response_auction_titles = auction_response.map {|auction| auction[:title]}
+    assert_equal 2, auction_response.length
+    assert_equal auctions_to_create.map {|auction| auction[:title]}, response_auction_titles
+  end
+
+  test 'should get /auctions/1 endpoint' do
```

Comment:
What is the difference between this test and the one right below it?

Reply:
When we created the  `/users` and `/users/:id` endpoints, I had a test to makes sure the endpoint responded with a 200 and a separate one to make sure we got the expected response. Is that redundant here?

Reply:
I'm gonna be real honest and say that looking at both tests again I'm not sure why I thought they were similar.